examples/ui.apps/src/content/jcr_root/apps/core-components-examples/components/version/VersionHelper.java

```diff
@@ -0,0 +1,39 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2021 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+import org.osgi.framework.FrameworkUtil;
+
+import com.adobe.cq.sightly.WCMUsePojo;
+import com.adobe.cq.wcm.core.components.models.Page;
+
+/**
+ * Helper Use-Object for displaying the Core Components bundle version
+ *
+ * Usage:
+ *    <sly data-sly-use.version="VersionHelper">${version.bundleVersion}</sly>
+ */
+public class VersionHelper extends WCMUsePojo {
```

Comment:
I'd rename it `CoreComponentBundle` as it is modelling the CC bundle and outputs its version.

Reply:
Done in f9f6fea